src/app/tests/suites/certification/Test_TC_TCTL_2_1.yaml

```diff
@@ -22,76 +21,88 @@ PICS:
-      disabled: true
+      command: "readAttribute"
+      attribute: "Step"
+      response:
+          value: StepValue
+          constraints:
+              type: temperature
+              maxValue: MaxTemperatureValue - MinTemperatureValue
+
+    - label:
+          "Verify that the DUT response (maxTemperature - minTemperature) % the
+          received value of the Step attribute == 0."
+      PICS: TCTL.S.A0003 && TCTL.S.A0002 && TCTL.S.A0001
+      cluster: "EqualityCommands"
+      command: "UnsignedNumberEquals"
+      arguments:
+          values:
+              - name: "Value1"
+                value: ( MaxTemperatureValue - MinTemperatureValue ) % StepValue
+              - name: "Value2"
+                value: 0
+      response:
+          - values:
+                - name: "Equals"
+                  value: true
+                  saveAs: IsExpectedValue
 
     - label: "Step 5: TH reads from the DUT the TemperatureSetpoint attribute"
       PICS: TCTL.S.A0000
-      verification: |
-          ./chip-tool temperaturecontrol read temperature-setpoint 1 1
-          On TH(chip-tool), Verify that the DUT response contains a temperature with value between minTemperature and maxTemperature inclusive.
-          and and (temperature value - minTemperature) % the received value of the Step attribute from step 4 == 0.
-
-          [1689770673.417838][9280:9282] CHIP:DMG:         SuppressResponse = true,
-          [1689770673.417845][9280:9282] CHIP:DMG:         InteractionModelRevision = 1
-          [1689770673.417851][9280:9282] CHIP:DMG: }
-          [1689770673.417920][9280:9282] CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0056 Attribute 0x0000_0000 DataVersion: 2844459846
-          [1689770673.417946][9280:9282] CHIP:TOO:   TemperatureSetpoint : 0
-          [1689770673.418017][9280:9282] CHIP:EM: <<< [E:21789i S:28147 M:181187483 (Ack:172880761)] (S) Msg TX to 1:0000000000000001 [9CA1] --- Type 0000:10 (SecureChannel:StandaloneAck)
-      disabled: true
+      command: "readAttribute"
+      attribute: "TemperatureSetpoint"
+      response:
+          saveAs: temperatureValue
+          constraints:
+              minValue: MinTemperatureValue
+              maxValue: MaxTemperatureValue
+
+    - label:
+          "Verify that the DUT response (temperature value - minTemperature) %
+          the received value of the Step attribute from step 4 == 0."
+      PICS: TCTL.S.A0000 && TCTL.S.A0001
+      cluster: "EqualityCommands"
+      command: "UnsignedNumberEquals"
+      arguments:
+          values:
+              - name: "Value1"
+                value: ( temperatureValue - MinTemperatureValue ) % 4
+              - name: "Value2"
+                value: 0
+      response:
+          - values:
+                - name: "Equals"
+                  value: true
+                  saveAs: IsExpectedValue
```

Comment:
Again, where is IsExpectedValue being used?  It's not.  It shouldn't be here.

Reply:
Removed.